sql/plan/insert.go

```diff
@@ -333,24 +240,27 @@ func (ii InsertInto) DebugString() string {
 	return pr.String()
 }
 
+// Expressions implements the sql.Expressioner interface.
 func (ii *InsertInto) Expressions() []sql.Expression {
 	return append(ii.OnDupExprs, ii.checks.ToExpressions()...)
 }
 
-func (ii InsertInto) WithExpressions(newExprs ...sql.Expression) (sql.Node, error) {
+// WithExpressions implements the sql.Expressioner interface.
+func (ii *InsertInto) WithExpressions(newExprs ...sql.Expression) (sql.Node, error) {
```

Comment:
Great change to use a pointer receiver consistently for all methods in the type!! 🙏